README.md

```diff
@@ -13,16 +13,36 @@ Built With
 
 ![Alt text](./src/media/demo.png?raw=true)
 
+### Prerequisites
 
+- Git/GitHub;
+- Optional ( IDE installed in your machine ) - recommended IDE: Vscode;
+- Node installed locally;
 
+### Setup
 
-To get a local copy up and running follow these simple example steps.
+- To clone this repo:
+  - Open the Terminal(Mac/Linux) or the Cmd (Windows);
+  - Navigate to where you want to paste the Repo folder;
+  - Type `git clone git@github.com:mateo951/math-magicians.git`;
+  - Open the folder todo-list that was created on your IDE;
 
-- On your terminal and run the following command to clone the repository
+### Install
 
-  `git@github.com:mateo951/math-magicians.git`
+- To install all the dependencies in the package-lock.json:
+  - Open the Terminal(Mac/Linux) or the Cmd (Windows);
+  - Navigate to be inside the math-magicians repo folder;
+  - Run npm install or npm i;
+  
+### Run tests
 
-#### To test the app locally, use Live Server (Extension for VS Code) to avoid js module CORS policy ####
+- Linters Tests:
+  - Eslint: run `npx eslint .`
+  - Stylelint: run `npx stylelint "**/*.{css,scss}"`
+- Run a Live Version of the Dist Folder:
+  - Run `npm start`
+- To Deploy to GH-Pages:
+  - Run `npm run deploy`
```

Comment:
- [ ] The most important test command has been omitted: `npm run test` or `npm test`. Please let's add it. :pray:

Reply:
Done